jaxrs/src/main/java/se/fortnox/reactivewizard/ExceptionHandler.java

```diff
@@ -96,6 +97,15 @@ public Publisher<Void> handleException(HttpServerRequest request, HttpServerResp
         return empty();
     }
 
+    private static boolean isAnticipatedClientException(Throwable throwable) {
+        return throwable instanceof ClosedChannelException
+            || throwable instanceof AbortedException
+            || throwable instanceof CancellationException
+            || (throwable instanceof IOException
+            && throwable.getMessage() != null
```

Comment:
What i think was the idea was to have `|| isBrokenPipe(throwable)` to increase the readability.